### PR TITLE
Set up data pipeline to find genes in the RMTL list

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

File: modules/RMTL.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    target    
+    RMTLList
+@:return target
+'''
+def addRMTLProperyToTarget(target,RMTLList):
+     for rmtl in RMTLList:
+         #  target id matchs Ensembl_ID
+         if rmtl[0]==target["id"]:
+             logger.info("target -- %s --- matches RMTL list %s ", target["id"], rmtl[2])
+             target["rmtl_approved_symbol"]=rmtl[1]
```
Comment (zdorman) [inline modules/RMTL.py:35]: rmtl_approved_symbol value probably not needed in the JSON, unless you want to keep it for a validation mechanism. 
It is the same as the existing Open Targets ApprovedSymbol value, and both are pulled using the Ensembl_ID. I kept the rmtl_approved_symbol field in the .csv only for validation

File: config.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+rmtl:
+  inputs:
+    rmtl: data/inputs/rmtl.csv
+    targets:
```
Comment (zdorman) [inline config.yaml:5]: Are these placeholder JSON locations? I'm thinking the production version will cycle through the list of 34 target folders in the database.

File: modules/RMTL.py (hunk 1 above)
Comment (zdorman) [inline modules/RMTL.py]: rmtl_approved_symbol value probably not needed in the JSON, unless you want to keep it for a validation mechanism. 
It is the same as the existing Open Targets ApprovedSymbol value, and both are pulled using the Ensembl_ID. I kept the rmtl_approved_symbol field in the .csv only for validation

File: modules/cfg.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+"""
+This will create a singleton argument parser that is appropriately configured
+with the various command line, environment, and ini/yaml file options.
+
+"""
+
+
+def setup_parser():
+    p = configargparse.get_argument_parser(config_file_parser_class=configargparse.YAMLConfigFileParser)
+    p.description = 'PPDC data piple line'
```
Comment (zdorman) [inline modules/cfg.py]: Did you mean "pipeline" instead of "piple line" ?

**Review by zdorman — COMMENTED** [review]
Looks good. Only suggested change is that the RMTL approved name field is redundant in the JSON, so is not needed in output.

File: config.yaml (hunk 2 above)
Comment (jonkiky) [reply config.yaml:5]: Yes, they are placeholders.  For the testing purpose, I only put one JSON in there.

File: modules/RMTL.py (hunk 1 above)
Comment (jonkiky) [reply modules/RMTL.py]: Will take rmtl_approved_symbol away from output

**Pull request merged**